plugins/azure_ad_admin/plugin.spec.yaml

```diff
@@ -23,6 +23,302 @@ hub_tags:
+      description: External User State Change Date Time
+      required: false
+    faxNumber:
+      title: Fax Number
+      type: string
+      description: Fax Number
+      required: false
+    givenName:
+      title: Given Name
+      type: string
+      description: Given Name
+      required: false
+    jobTitle:
+      title: Job Title
+      type: string
+      description: Job Title
+      required: false
+    legalAgeGroupClassification:
+      title: Legal Age Group Classification
+      type: string
+      description: Legal Age Group Classification
+      required: false
+    mail:
+      title: Mail
+      type: string
+      description: Mail
+      required: false
+    mailNickname:
+      title: Mail Nickname
+      type: string
+      description: Mail Nickname
+      required: false
+    mobilePhone:
+      title: Mobile Phone
+      type: integer
+      description: Mobile Phone
+      required: false
+    onPremisesDistinguishedName:
+      title: On Premises Distinguished Name
+      type: string
+      description: On Premises Distinguished Name
+      required: false
+    onPremisesDomainName:
+      title: On Premises Domain Name
+      type: string
+      description: On Premises Domain Name
+      required: false
+    onPremisesImmutableId:
+      title: On Premises Immutable ID
+      type: string
+      description: On Premises Immutable ID
+      required: false
+    onPremisesLastSyncDateTime:
+      title: On Premises Last Sync Date Time
+      type: date
+      description: On Premises Last Sync Date Time
+      required: false
+    onPremisesSecurityIdentifier:
+      title: On Premises Security Identifier
+      type: string
+      description: On Premises Security Identifier
+      required: false
+    onPremisesSamAccountName:
+      title: On Premises Sam Account Name
+      type: string
+      description: On Premises Sam Account Name
+      required: false
+    onPremisesSyncEnabled:
+      title: On Premises Sync Enabled
+      type: string
+      description: On Premises Sync Enabled
+      required: false
+    onPremisesUserPrincipalName:
+      title: On Premises User Principal Name
+      type: string
+      description: On Premises User Principal Name
+      required: false
+    otherMails:
+      title: Other Mails
+      type: "[]string"
+      description: Other Mails
+      required: false
+    passwordPolicies:
+      title: Password Policies
+      type: string
+      description: Password Policies
+      required: false
+    officeLocation:
+      title: Office Location
+      type: string
+      description: Office Location
+      required: false
+    postalCode:
+      title: Postal Code
+      type: string
+      description: Postal Code
+      required: false
+    preferredDataLocation:
+      title: Preferred Data Location
+      type: string
+      description: Preferred Data Location
+      required: false
+    proxyAddresses:
+      title: Proxy Addresses
+      type: array
```

Comment:
Should this be updated to an array of type, for instance `[]string`? Repeated below.